zebedee-cms/src/test/java/com/github/onsdigital/zebedee/ZebedeeTest.java

```diff
@@ -18,200 +20,408 @@
 
 import static com.github.onsdigital.zebedee.Builder.COLLECTION_ONE_NAME;
 import static com.github.onsdigital.zebedee.Builder.COLLECTION_TWO_NAME;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.core.Is.is;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertThrows;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyZeroInteractions;
+import static org.mockito.Mockito.when;
 
 
 public class ZebedeeTest extends ZebedeeTestBaseFixture {
 
-	Path expectedPath;
-	Map<String, String> env;
+    Path expectedPath;
+    Map<String, String> env;
 
-	public void setUp() throws Exception {
-		env = Root.env;
-		Root.env = new HashMap<>();
-		expectedPath = builder.parent;
-	}
+    public void setUp() throws Exception {
+        MockitoAnnotations.initMocks(this);
 
-	@Override
-	public void tearDown() throws Exception {
-		// do nothing.
-	}
+        env = Root.env;
```

Comment:
All changes from line `1 to 228` are updates to the file formatting (replaced tabs with spaced). Jump to next comment (line `229`) for start of code changes.